packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -114,17 +114,16 @@
     &.is-incomplete {
       background-color: var(--terra-time-input-incomplete-background-color, #ffc);
 
-      &:hover,
-      .desktop:hover {
+      &:hover {
         background-color: var(--terra-time-input-incomplete-hover-background-color, #ffc);
       }
 
       &:focus {
         background-color: var(--terra-time-input-incomplete-focus-background-color, #ffc);
       }
 
-      .desktop.initial-focus {
-        background-color: var(--terra-time-input-desktop-incomplete-focused-input-background-color, rgb(185, 215, 233));
+      .initial-focus {
+        background-color: var(--terra-time-input-incomplete-focused-input-background-color, rgb(185, 215, 233));
```

Comment:
Don't forget to update the various themes this variable change impacts.

Reply:
@avinashg1994 This needs to be accounted for as part of https://github.com/cerner/terra-framework/pull/978 and https://github.com/cerner/terra-framework/pull/975.